create_task_files.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+func createDirectory(dir string) error {
+	// Create new directory
+	if err := os.Mkdir(dir, 0755); err != nil {
+		return err
+	}
+	return nil
+}
+
+func createFile(file string) error {
+	if _, err := os.Create(file); err != nil {
+		return err
+	}
+	return nil
+}
+
+// Checks that a given task is as expected
+func createTaskFiles(t task) (checkResult, error) {
+	// Checks if in a git repo
+	out, err := exec.Command("git", "rev-parse", "--show-toplevel").Output()
+	if err != nil {
+		message := fmt.Sprintf("Current directory does not seem to be a git repository.\n")
+		return checkResult{Passed: false, Message: message}, nil
+	}
+
+	// Checks for the specified repo
+	if t.GithubRepo != "" {
+		// Obtains the name of current repo
+		repoPath := strings.Split(string(out), "/")
+		depth := len(repoPath) - 1
+		repo := strings.TrimRight(repoPath[depth], "\n")
+
+		// Determines if run from correct repository
+		if repo != t.GithubRepo {
+			message := fmt.Sprintf("No remote pointing to a \"%s\" Github repository; make sure to run this application from your project's directory.", t.GithubRepo)
```

Comment:
I think you're checking the name of the local folder, which can be different to the name of the remote; and the project constraint is about the name of the remote (we don't care if the student put the code in `~/llp`, as long as the repo's URL is `https://github.com/<user>/holbertonschool-lower_level_programming`).
I wrote the bit to do just that (the bit in `main()` that runs `git remote -v`), you should probably factorize it into a function, and use it here.
More generally, feel free to kick some of my code out of `main()` if it makes sense; I put it there because there was no reason to factorize, but now that we have two features with some overlap, there definitely are reasons.